ci/mutmut_report.py

```diff
@@ -0,0 +1,50 @@
+import subprocess
+import json
+import os
+import csv
+
```

Comment:
As I can't leave comments to unchanged lines and to files in general, noting in random place.
General note for future:
I feel that some real output of mutmut might be of use. Now it tells us nothing but the rate, there is no output in the build. Suppose we have large survival rate and want to check some of the mutants. Would we be required to rerun mutmut locally? Or maybe mutmut stores the results somewhere so we can read them and store in permanent location?

Reply:
Yes, I recommend that we run mutmut locally to get a proper workflow. The badge is intended to be a gentle nudge to start using mutation testing.